data/deploy.rb

```diff
@@ -17,6 +18,9 @@
 set :deploy_to, lambda { "/home/#{settings.user!}/preflight/#{settings.app_name!}-#{settings.rails_env!}" }
 set :repository, ENV['REPOSITORY']
 set :branch, ENV['BRANCH']
+set :remove_git_dir, false
```

Comment:
This adds a configuration flag to control: https://github.com/mina-deploy/mina/pull/348